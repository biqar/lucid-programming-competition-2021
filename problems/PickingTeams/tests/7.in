19
? Alice Bob
? Bob Charlie
? Charlie Alice
? David Emily
? Emily Francis
? Ginny Helene
? Irene James
? James Irene
? James Alice
? Kyle Bob
? Charlie Helene
? Ginny Bob
? Bob Ginny
? Emily Kyle
? Alice Bob
? Alice Ginny
? Lily Matthew
? James David
? Kyle David
