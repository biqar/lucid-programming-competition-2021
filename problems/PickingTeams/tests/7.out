no
no
no
no
no
no
no
no
no
no
no
no
no
no
no
no
no
no
no
13
